src/documentation/domains/webhook/controller/webhook.controller.docs.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @swagger
+ * paths:
+ *   /api/webhooks/{webhookId}:
+ *     post:
+ *       summary: Receives webhooks
+ *       description: This endpoint receives webhooks from external services.
+ *       tags:
+ *         - Webhook
+ *       parameters:
+ *         - in: path
+ *           name: webhookId
+ *           required: true
+ *           type: string
+ *           description: The unique identifier of the registered webhook.
+ *       responses:
+ *         '200':
```

Comment:
As a comment, remember to add in the future the returned Schema and an example of it. In case you decide not to send any DTO you may change your HTTP respoonse number

Reply:
I decided to not return anything based on the [Linear Docs](https://developers.linear.app/docs/graphql/webhooks#how-does-a-webhook-work). Based on that document, a 200 HTTP code in response to a successful request is enough. Besides, I think you wouldn't want to attach a body to that response, it will not be processed by the client server and you'll be exposing unnecessary data.